src/function/table/read_wal.cpp

```diff
@@ -0,0 +1,112 @@
+#include "duckdb/common/types.hpp"
+#include "duckdb/function/table/range.hpp"
+#include "duckdb/function/function_set.hpp"
+#include "duckdb/main/client_context.hpp"
+#include "duckdb/storage/wal_reader.hpp"
+#include "duckdb/main/database_manager.hpp"
+//#include "json_serializer.hpp"
```

Comment:
@Tishj @Mytherin it's not clear how to use the JSON serializer functionality here. It's in an extension.

Can we address that in another PR after this one is merged? If so, I'll remove the commented out code here and a few lines below.